ophyd/userapi/scan_api.py

```diff
@@ -17,7 +17,7 @@
 session_manager = get_session_manager()
 logger = session_manager._logger
 
-__all__ = ['AScan', 'DScan', 'Scan', 'Data', 'Count']
+__all__ = ['AScan', 'DScan', 'Data', 'Count']
```

Comment:
sitting with @dchabot and @danielballan at XPD. dan and daron discussed it and I simply added the change to this PR